README.md

```diff
@@ -392,6 +398,7 @@ $ atlantis server --atlantis-url $URL --gitlab-user $USERNAME --gitlab-token $TO
 - `$USERNAME` is the GitHub/GitLab username you generated the token for
 - `$TOKEN` is the access token you created. If you don't want this to be passed in as an argument for security reasons you can specify it in a config file (see [Configuration](#configuration)) or as an environment variable: `ATLANTIS_GH_TOKEN` or `ATLANTIS_GITLAB_TOKEN`
 - `$SECRET` is the random key you used for the webhook secret. If you left the secret blank then don't specify this flag. If you don't want this to be passed in as an argument for security reasons you can specify it in a config file (see [Configuration](#configuration)) or as an environment variable: `ATLANTIS_GH_WEBHOOK_SECRET` or `ATLANTIS_GITLAB_WEBHOOK_SECRET`
+- `$GITHUBHOSTNAME` is the URL of your enterprise Github
```

Comment:
can you give an example here? So people know if it should contain the scheme or not (it shouldn't). Like:
... is the URL of your enterprise GitHub, ex. github.mycompany.com (don't add https://).

Reply:
Done